tsl/test/expected/continuous_aggs_usage.out

```diff
@@ -102,6 +102,10 @@ last_run_started_at    |
 job_status             | 
 last_run_duration      | 
 next_scheduled_run     | 
+total_runs             | 
+total_successes        | 
+total_failures         | 
+total_crashes          | 
```

Comment:
Is it possible, or do you think it would be helpful - to also show last successful run? or is that overlapping with completed_threshold?

Reply:
The catalog tables show the status of the last job last time - did it succeed or fail? We  don't have enough info to pin point when was the last time  the job succeeded. Is that what  you are looking for?